Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Makie"
 uuid = "ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a"
 authors = ["Simon Danisch", "Julius Krumbiegel"]
-version = "0.20.7"
+version = "0.20.8"
```

Comment:
```suggestion
version = "0.20.7"
```
We'll have to update the version later, since all the backends run in exact sync with the Makie version.

Reply:
Thanks!

Reply:
```suggestion
version = "0.20.7"
```